storage/src/block_store/lmdb/versioned_databases.rs

```diff
@@ -267,6 +275,7 @@ where
 
     /// Returns the keys from the `current` database only.
     #[cfg(test)]
+    #[allow(dead_code)]
```

Comment:
Looks like this method can be deleted now?

Reply:
removed in https://github.com/casper-network/casper-node/pull/4498/commits/1a0f5e3ea966156ee54ade5b2015268c5217598a